docxcompose/properties.py

```diff
@@ -69,6 +69,11 @@ def vt2value(element):
         return element.text
 
 
+def is_text_property(property):
+    tag = QName(property).localname
+    return tag in ('bstr', 'lpstr', 'lpwstr')
```

Comment:
Semantically this should be a list, not a tuple.
See: https://stackoverflow.com/questions/626759/whats-the-difference-between-lists-and-tuples